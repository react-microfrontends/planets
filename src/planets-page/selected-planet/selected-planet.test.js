import SelectedPlanet from "./selected-planet.component.js";
import React from "react";
import renderer from "react-test-renderer";

describe("selected-planet", () => {
  it("should render when no planet is selected", () => {
    const tree = renderer.create(<SelectedPlanet />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
