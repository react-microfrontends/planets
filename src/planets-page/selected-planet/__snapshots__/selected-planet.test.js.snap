// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`selected-planet should render when no planet is selected 1`] = `
<div>
  No planet Selected
</div>
`;
